modules/local/rename_reads.nf

```diff
@@ -0,0 +1,46 @@
+process RENAME_READS {
+    tag "$meta.id"
+    label "process_single"
+
+    conda "conda-forge::sed=4.7"
+    if (workflow.containerEngine == 'singularity' && !tast.ext.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/ubuntu:20.04"
+    } else {
+        container "registry.hub.docker.com/biocontainers/biocontainers:v1.2.0_cv2"
+    }
+
```

Comment:
I think we can do better than that, but not blocking for the release

Reply:
Can we bump this into an issue and deal with it post-release?